### PR TITLE
Search text

### PR DESCRIPTION
**Pull request by nevogithub**

### CODE REVIEW

**Comment by nevogithub** [comment]
It has some search textss and users

File: .vscode/launch.json
Comment (YechiamW3) [inline .vscode/launch.json:1]: shouldn't this folder be in gitignore?

File: cv_next/.env
Comment (YechiamW3) [inline cv_next/.env:1]: Pretty sure this shouldn't be pushed. @Aradon267?

File: cv_next/server/api/cvs.ts [hunk 1]
```diff
@@ -164,7 +169,10 @@ export async function getPaginatedCvs(
     if (filterOutDeleted) {
       query = query.eq("deleted", false);
     }
-
+    if(checkNames){
+      ids = await queryUserId;
```
Comment (YechiamW3) [inline cv_next/server/api/cvs.ts:173]: Is this working?
I think it should be `{ data: ids }`...

Comment (YechiamW3) [reply cv_next/server/api/cvs.ts:173]: Oh I see you simply don't extract data from the result; it's preferable to do so. See the other functions. Lets you early-exit if you encounter an error.

File: .vscode/launch.json
Comment (Aradon267) [reply .vscode/launch.json:1]: This should

File: cv_next/.env
Comment (Aradon267) [reply cv_next/.env:1]: Same for that yes

File: cv_next/server/api/cvs.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import MyLogger from "@/server/base/logger";
 import Categories from "@/types/models/categories";
 import Definitions from "../../lib/definitions";
 import SupabaseHelper from "./supabaseHelper";
-import { PostgrestError } from "@supabase/supabase-js";
+import { PostgrestError, } from "@supabase/supabase-js";
```
Comment (PeroG28) [inline cv_next/server/api/cvs.ts:7]: delete, this shold deleted by the formater (if we have one)

File: cv_next/server/api/cvs.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import MyLogger from "@/server/base/logger";
...
 import { filterValues } from "@/app/feed/components/filterPanel";
 
 export const revalidate = Definitions.CVS_REVALIDATE_TIME_IN_SECONDS;
```
Comment (PeroG28) [inline cv_next/server/api/cvs.ts:10]: REVALID_DATE, global  const needs to be capitel with _ between words

File: cv_next/server/api/cvs.ts [hunk 4]
```diff
@@ -147,13 +147,18 @@ export async function getPaginatedCvs(
     const supabase = SupabaseHelper.getSupabaseInstance();
     let query = supabase
```
Comment (PeroG28) [inline cv_next/server/api/cvs.ts:148]: why this is not one query with join?

File: cv_next/server/api/cvs.ts [hunk 5]
```diff
@@ -147,13 +147,18 @@ export async function getPaginatedCvs(
...
+      .select("*");
+    let queryUserId = supabase
+      .from("profiles")
+      .select("*");
+    let checkNames = false;
+    let ids = null;
     MyLogger.logDebug("filters", filters);
     if (filters) {
       if (filters.searchValue) {
         query = query.textSearch("description", filters.searchValue);
```
Comment (PeroG28) [inline cv_next/server/api/cvs.ts:159]: try to combine queries, maybe there is a way to text search 2 fileds

File: cv_next/server/api/cvs.ts (hunk 3 above)
Comment (Aradon267) [reply cv_next/server/api/cvs.ts:10]: This isn't used so can be deleted
and revalidate is one word

File: cv_next/server/api/cvs.ts (hunk 1 above)
Comment (nevogithub) [reply cv_next/server/api/cvs.ts:173]: Yeah I'll fix it to match the error handling pattern

File: cv_next/server/api/cvs.ts (hunk 4 above)
Comment (nevogithub) [reply cv_next/server/api/cvs.ts:148]: IMHO it should be a seperate query as we;re querying different data. I've been working on a function to query IDs from fullnames seperately

**Comment by Aradon267** [comment]
@nevogithub is everything alright with this? sent you on WhatsApp the stable filter version as we discussed

**Comment by Aradon267** [comment]
As discussed in the last weekly, I'll handle things here

**Pull request closed without merging**